data_pipeline/test_dp02_speed_clusters.py

```diff
@@ -12,36 +12,39 @@
 from pathlib import Path
 
 import dp02_speed_clusters
-import dummy_competition_setup_for_testing
 import numpy as np
 from dp02_speed_clusters import main
-from dummy_competition_setup_for_testing import create_dummy_competition_setup
 from h5_helpers import load_h5_file
 
+import t4c22
+from t4c22.misc.dummy_competition_setup_for_testing import create_dummy_competition_setup
+
 
 def test_speed_clusters():
     dates = ["1970-01-01", "1970-01-02", "1970-01-03"]
     with tempfile.TemporaryDirectory() as tmp_dir:
         tmp_dir = Path(tmp_dir)
 
-        dp02_speed_clusters.NUM_SLOTS = dummy_competition_setup_for_testing.NUM_SLOTS_AGGREGATED
-        dp02_speed_clusters.NUM_ROWS = dummy_competition_setup_for_testing.NUM_ROWS
-        dp02_speed_clusters.NUM_COLUMNS = dummy_competition_setup_for_testing.NUM_COLUMNS
+        dp02_speed_clusters.NUM_SLOTS = t4c22.misc.dummy_competition_setup_for_testing.NUM_SLOTS_AGGREGATED
+        dp02_speed_clusters.NUM_ROWS = t4c22.misc.dummy_competition_setup_for_testing.NUM_ROWS
+        dp02_speed_clusters.NUM_COLUMNS = t4c22.misc.dummy_competition_setup_for_testing.NUM_COLUMNS
 
         create_dummy_competition_setup(basedir=tmp_dir, city="london", skip_train_labels=True, train_dates=dates, skip_movie_15min=False)
         for date in dates:
             assert (tmp_dir / "movie_15min" / "london" / f"{date}_london_8ch_15min.h5").exists()
 
         speed_clusters_h5 = tmp_dir / "movie_speed_clusters" / "london" / f"speed_clusters.h5"
         assert not speed_clusters_h5.exists()
+        heatmap_h5 = tmp_dir / "movie_heatmap" / "london" / f"probe_heatmap.h5"
+        assert not heatmap_h5.exists()
 
         main(["-d", str(tmp_dir), "-c", "london", "-n", "3"])
 
         assert speed_clusters_h5.exists()
         speed_clusters_data = load_h5_file(speed_clusters_h5)
         assert speed_clusters_data.shape == (
-            dummy_competition_setup_for_testing.NUM_ROWS,
-            dummy_competition_setup_for_testing.NUM_COLUMNS,
+            t4c22.misc.dummy_competition_setup_for_testing.NUM_ROWS,
```

Comment:
@moritz9 add t4c22 as dependency in `environment.yaml` or install for running tests in github or exclude from tests in github?

Reply:
Fixed